omnigan/data.py

```diff
@@ -81,6 +81,11 @@ def pil_image_loader(path, task):
         arr = arr.astype(np.float32)
         arr[arr != 0] = 1 / arr[arr != 0]
 
+    if task == 'm':
+        arr[arr != 0] = 1 / arr[arr != 0]
+        #Make sure mask is RGB for the sake of transforms
+        if len(arr.shape) >= 3:
+            arr = arr[:,:,0]
```

Comment:
how does that ensure rgb?

Reply:
It does the opposite actually haha. I'll fix the comment